src/cpu.rs

```diff
@@ -238,6 +269,12 @@ pub(crate) mod arm {
 
             #[cfg(all(
                 any(target_os = "android", target_os = "fuchsia", target_os = "linux"),
```

Comment:
Shouldn't this line get deleted? The new block `any` block right under contains all the same oses, + windows.